styles/_colors.scss

```diff
@@ -0,0 +1,5 @@
+$color-black: #000000cc;
+$color-green: #6ab04c;
+$color-white-grey: #dddddd;
+$color-white: white;
+$color-black-transparent: #000000ea;
```

Comment:
in this case AND ONLY WITH TRANSPARENCY you can use rgba()

`rgba($black, 0.5)`